Tab swaps to dragging the opposite handle

DaraghD opened the pull request:
Unsure of naming of the function

Partially closes #1870

0HyperCube commented:
!build

0HyperCube reviewed (commented):
The code looks good so far and works well.

Currently if both the handles from an anchor are selected, <kbd>tab</kbd> switches to just having a random one selected. Is this the desired behaviour? I would have thought it would be more intuitive just to keep both handles selected (since they are both opposite to a formerly selected handle). However it probably doesn't matter too much.

Also please can you add a hint for this. These are the text at the bottom of the viewport:
![screenshot with hint in](https://github.com/user-attachments/assets/5a10ac54-7c50-4020-a8f8-bbe5d6108398)

Thanks for your work on this contribution so far.

DaraghD commented:
> Currently if both the handles from an anchor are selected, tab switches to just having a random one selected. Is this the desired behaviour? I would have thought it would be more intuitive just to keep both handles selected (since they are both opposite to a formerly selected handle). However it probably doesn't matter too much.

Switched it to be that way, I think that makes more sense

> Also please can you add a hint for this. These are the text at the bottom of the viewport: 

Added one, again not sure of the naming

0HyperCube reviewed (approved):
Works well and the code looks good. Thanks again for working on this.

DaraghD commented:
Clippy failed on a ton of stuff that I didn't touch, is that normal?

0HyperCube commented:
@DaraghD the clippy lints are fixed by https://github.com/GraphiteEditor/Graphite/commit/a395fbf0637c23b8b05a7c451ff7b8421587b655.

Keavon commented:
I'll do a QA review on this before we merge it. It's in my queue that I'm working through presently.

Keavon commented:
@DaraghD upon testing this, I think the best near-term solution for the question of cursor alignment is to enable cursor lock so the user's cursor disappears, but don't yet attempt to render a fake cursor. Could you please add that part? You can reference how we do this in the NumberInput.svelte slider drag, for example. That's all frontend though, and this'll need to plumbed in from the backend. You could also look at the code paths which let the backend plumb to the frontend which cursor style to display, since this would be similar and could potentially be included in those code paths. Thanks!

Keavon commented:
!build

Keavon commented:
This will be ready to merge once GitHub fixes their CI outage and I can make it re-run to pass CI, which should be when I wake up.
